Project.toml

```diff
@@ -15,7 +16,6 @@ Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 
 [compat]
-Gridap = "0.15"
```

Comment:
Please, do not remove Gridap from the compat section. Use Gridap 0.15.3 since this is the version that has the bugfix we need.

Reply:
Ok, Added. But I was not aware that I had removed it ... perhaps it was the package manager automatically or something like that?